### DISCUSSION TITLE
Garbo stuck on shadows/obsessed with shadow waters

### DISCUSSION THREAD

**Discussion by Vecingettorix** [loathers/garbage-collector#1869] [Q&A] [answered]
I keep encountering a problem with when running garbo with the closed-phone interactions.
So far I have had the issue as a seal clubber, pastamancer, sauceror, and disco bandit. I have saucegeyser permed.
Problems;
1. garbo often gets stuck against the shadow monsters and will decide not to use saucegeyser. This doesn't seem to be a mana issue or anything wrong with my character as when i take manual control in the relay browser i just use saucegeyser a couple of times until the monster is dead.
2. (most infuriating of these) garbo seems obsessed with getting a seemingly arbitrary excess number of adventures of Shadow Water. It will regularly still be adventuring in the shadow rifts when i already have 100 more adventures of the effect than adventures to actually use!
Any help very welcome.

thanks.

**Comment by Rinn** [comment, marked as answer]
for 1 i made a pretty large fix in the combat macros last month, have you noticed any issues since mid may? https://github.com/loathers/garbage-collector/commit/71abf5e114b822fd1293c8efd2c208190964e18b

**Reply by Vecingettorix** [reply]
Hey. Thanks, it seems to be running much better now!
